padl/transforms.py

```diff
@@ -378,6 +392,31 @@ def pd_zip_save(self, path: Union[Path, str], force_overwrite: bool = False):
                     if file.is_file():
                         zipf.write(file, file.name)
 
+    @classmethod
+    def _pd_process_options(cls, options):
+        """
```

Comment:
For consistency with the rest of the package, please don't put a newline to the top of a docstring.